src/ui/components/ShowMoreCard/index.js

```diff
@@ -44,43 +44,43 @@ const initialUIState: UIStateType = {
 export class ShowMoreCardBase extends React.Component<InternalProps> {
   contents: HTMLElement | null;
 
-  componentWillReceiveProps(nextProps: InternalProps) {
-    const { children } = this.props;
-    const { children: newChildren } = nextProps;
+  componentDidUpdate(prevProps: InternalProps) {
+    const { children: prevChildren } = prevProps;
+    const { children, uiState } = this.props;
 
     let html =
+      prevChildren.props &&
+      prevChildren.props.dangerouslySetInnerHTML &&
+      prevChildren.props.dangerouslySetInnerHTML.__html;
+
+    let newHtml =
```

Comment:
Nit: Because we've changed from `componentWillReceiveProps` to `componentDidUpdate`, in the context of this method we're really dealing with `html` and `oldHtml`, rather than `html` and `newHtml`. It might be clearer if you changed the names and switched them around.

Reply:
ah right, thanks!